Remaps/id/PSPE/pe-item-mapping.js

```diff
@@ -0,0 +1,3640 @@
+	registerItemRemap(43, 6, 43, 7);
+	// Prismarine data ID mismatch
+	registerItemRemap(168, 1, 168, 2);
+	registerItemRemap(168, 2, 168, 1);
+	// Podzol
+	registerItemRemap(3, 2, 243, 0);
+	// Colored Fences
+	registerItemRemap(188, 0, 85, 1);
+	registerItemRemap(189, 0, 85, 2);
+	registerItemRemap(190, 0, 85, 3);
+	registerItemRemap(192, 0, 85, 4);
+	registerItemRemap(191, 0, 85, 5);
+	// Shulker Boxes
+	registerItemRemap(219, 218, 0); // WHITE_SHULKER_BOX
+	registerItemRemap(220, 218, 1); // ORANGE_SHULKER_BOX
+	registerItemRemap(221, 218, 2); // MAGENTA_SHULKER_BOX
+	registerItemRemap(222, 218, 3); // LIGHT_BLUE_SHULKER_BOX
+	registerItemRemap(223, 218, 4); // YELLOW_SHULKER_BOX
+	registerItemRemap(224, 218, 5); // LIME_SHULKER_BOX
+	registerItemRemap(225, 218, 6); // PINK_SHULKER_BOX
+	registerItemRemap(226, 218, 7); // GRAY_SHULKER_BOX
+	registerItemRemap(227, 218, 8); // SILVER_SHULKER_BOX
+	registerItemRemap(228, 218, 9); // CYAN_SHULKER_BOX
+	registerItemRemap(229, 218, 10); // PURPLE_SHULKER_BOX
+	registerItemRemap(230, 218, 11); // BLUE_SHULKER_BOX
+	registerItemRemap(231, 218, 12); // BROWN_SHULKER_BOX
+	registerItemRemap(232, 218, 13); // GREEN_SHULKER_BOX
+	registerItemRemap(233, 218, 14); // RED_SHULKER_BOX
+	registerItemRemap(234, 218, 15); // BLACK_SHULKER_BOX
+
+	// ===[ ITEMS ]===
+	registerItemRemap(410, 422); // PRISMARINE_CRYSTALS
+	registerItemRemap(416, 425); // ARMOR_STAND
+	registerItemRemap(425, 446); // BANNER
+	registerItemRemap(434, 457); // BEETROOT
+	registerItemRemap(435, 458); // BEETROOT_SEEDS
+	registerItemRemap(436, 459); // BEETROOT_SOUP
+	registerItemRemap(443, 444); // ELYTRA
+	registerItemRemap(449, 450); // TOTEM
+	registerItemRemap(450, 445); // SHULKER_SHELL
+	registerItemRemap(322, 1, 466, 0); // Enchanted Golden Apple
+	registerItemRemap(444, 333, 1); // Spruce Boat
+	registerItemRemap(445, 333, 2); // Birch Boat
+	registerItemRemap(446, 333, 3); // Jungle Boat
+	registerItemRemap(447, 333, 4); // Acacia Boat
+	registerItemRemap(448, 333, 5); // Dark Oak Boat
+	registerItemRemap(422, 443); // Minecart with a Command Block
+	registerItemRemap(335, 325, 1); // Milk Bucket
+	registerItemRemap(326, 325, 8); // Water Bucket
+	registerItemRemap(327, 325, 10); // Lava Bucket
+	// Records
+	registerItemRemap(2256, 500);
+	registerItemRemap(2257, 501);
+	registerItemRemap(2258, 502);
+	registerItemRemap(2259, 503);
+	registerItemRemap(2260, 504);
+	registerItemRemap(2261, 505);
+	registerItemRemap(2262, 506);
+	registerItemRemap(2263, 507);
+	registerItemRemap(2264, 508);
+	registerItemRemap(2265, 509);
+	registerItemRemap(2266, 510);
+	registerItemRemap(2267, 511);
+
+	// Not implemented (yet) in PE
+	registerItemRemap(453, 340); // KNOWLEDGE BOOK -> BOOK
+	registerItemRemap(442, 268); // SHIELD -> WOODEN SWORD
+	registerItemRemap(439, 262); // SPECTRAL ARROW -> ARROW
+	registerItemRemap(343, 408); // POWERED MINECART -> MINECART WITH A HOPPER
+}
+
+function getInitialMapping() {
+	return [
```

Comment:
you could load this in via loading the json from raw github web, but this is a remap script, what do we care 😄